adds .taprc file support #228

kgraves opened the pull request.

kgraves committed:
adds .taprc file support #228

isaacs commented on bin/run.js:
```diff
@@ -93,26 +107,37 @@ function main () {
+  }
+
+  if (process.env.TAP === '1') {
+    defaultArgs.reporter = 'tap'
```
This should actually be set on the final option object, not just the default set.

isaacs commented on test/utils.js:
```diff
@@ -0,0 +1,52 @@
+      coverageReport: false,
+      reporter: 'classic'
+    }
+    t.strictSame(utils.parseRcFile(validYmlPath), expected)
```
Should this be checking a file that's not valid yaml?  Maybe use `__filename` or something?

kgraves committed:
a few changes to .taprc file support #234

includes:
- moves parsing of `process.env.TAP` back to where it was
- restructures dotfile tests
- updates a test that was copied from another, but not updated

isaacs commented:
Landed and published on 6.0.0.  thanks!

kgraves commented:
woohoo :+1:
